src/docs/product/performance/resources.md

```diff
@@ -0,0 +1,122 @@
+- The `resource.render_blocking_status` span data value is set to the [render blocking status](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/renderBlockingStatus) of the resource.
+
+Here's an example snippet of creating a resource span manually with the SDK.
+
+```js
+// https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming
+const resources = performance.getEntriesByType("resource");
+resources.forEach((entry) => {
+  const startTime = msToSec(entry.startTime);
+  const duration = msToSec(entry.duration);
+
+  const span = Sentry.startInactiveSpan({
+    startTimestamp: startTime,
+    op: entry.initiatorType
+      ? `resource.${entry.initiatorType}`
+      : "resource.other",
+    description: entry.name,
+    data: {
+      "http.response_transfer_size": entry.transferSize,
+      "http.response_content_length": entry.encodedBodySize,
+      "http.decoded_response_content_length": entry.decodedBodySize,
+      "resource.render_blocking_status": entry.renderBlockingStatus,
+    },
+  });
+
+  // override end timestamp to match performance entry
+  span.finish(startTime + duration);
+});
+```
+
+## Resources Page
+
+The **Resources** page gives you a quick overview of your application's resource performance for the selected project(s). You can use this page as a starting point to investigate potential problem resources and drill down to better understand how various resources are affecting your app's performance.
+
+Open the **Resources** page by clicking "Resources" in the sidebar, under "Performance". Alternatively, the **Performance** page also surfaces the highest impact queries in the [TBD] widget. From there, you can click "View All" to open the **Resources** page.
+
+At the top of the page, summary graphs for requests per minute (throughput) and average duration provide high-level insight into the performance of your resources. If you see an anomaly or want to investigate a time range further, you can click and drag to select a range directly in a graph to filter data for that time range.
+
+The resource table below shows a list of grouped resources, along with their type, their volume (requests per min), average duration, the total time your app spent loading that resource (time spent), and [average encoded size](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/encodedBodySize) of the resource.
+
+Currently only JavaScript and CSS resources are supported.
+
+<!-- TODO: Add information about time spent. Extract time spent docs into separate page? -->
+<!-- This paragraph also exists in queries table -->
+<!-- By default, this table is sorted by most [time spent](#what-is-time-spent). This means that queries at the top are usually really slow, very frequent, or both. You can change this to sort by queries per minute to see the most frequently run queries, or by average duration to see the slowest queries. -->
```

Comment:
@vivianyentran Do you have any opinions on this?
Rn we have this doc https://docs.sentry.io/product/performance/queries/#what-is-time-spent  for queries, but resources now has time spent as well.

I think we can either extract the time spent explanation into it's own section, share it between resources and queries and future modules.
Or we can just create a copy of it on the resource page (for now), and provide a resource specific example.

Reply:
I think extracting it into a reusable section could make sense if it's something we see more future modules including. For this, I don't think it's necessary to make it a whole section; we could just add a paragraph here to explain how it's calculated. Something like this:

> By default, the resources table is sorted by most [time spent](#what-is-time-spent), which serves as a proxy for the relative performance impact of a given resource. A resource's time spent is the sum of all its durations in a given time period or, put another way, the product of its average duration and requests per minute. This means that resources at the top are usually loading really slowly, very frequently, or both. 
> 
> You can click on a column header to change how the table is sorted. Sort by requests per minute to see the most frequently loaded resources or by average duration to see the slowest-loading resources.

Reply:
👍 Thanks for the input, i'm good with that idea, will add the paragraph